src/_articles/archive/zones.md

```diff
@@ -21,58 +21,49 @@ obsolete: true
-with a focus on the top-level `runZoned()` function.
+This article discusses zone-related APIs in the [dart:async][] library,
+with a focus on the top-level [`runZoned()`][]
+and [`runZonedGuarded()`][] functions.
 Before reading this article,
 you should be familiar with the techniques covered in
 [Futures and Error Handling](/guides/libraries/futures-error-handling).
 
-Currently, the most common use of zones is
-to handle errors raised in asynchronously executed code.
-For example,
-a simple HTTP server
-might use the following code:
-
-{% prettify dart tag=pre+code %}
-[!runZoned(() {!]
-  HttpServer.bind('0.0.0.0', port).then((server) {
-    server.listen(staticFiles.serveRequest);
-  });
-[!},
-onError: (e, stackTrace) => print('Oh noes! $e $stackTrace'));!]
-{% endprettify %}
-
-Running the HTTP server in a zone
-enables the app to continue running despite uncaught (but non-fatal)
-errors in the server's asynchronous code.
-
-{{site.alert.info}}
-  **API note:**
-  This use case won't always require zones.
-  We expect that isolates will have an API enabling you to
-  listen for uncaught errors.
-{{site.alert.end}}
+[dart:async]: ({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/dart-async-library.html)
+[`runZoned()`]: ({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/runZoned.html)
+[`runZonedGuarded()`]: ({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/runZonedGuarded.html)
 
 Zones make the following tasks possible:
 
-* Protecting your app from exiting due to
-  an uncaught exception thrown by asynchronous code,
-  as shown in the preceding example.
-
-* Associating data—known as
-  <em>zone-local values</em>—with individual zones.
+* **Protecting your app from exiting due to
+  an uncaught exception**.
+  For example,
+  a simple HTTP server
+  might use the following asynchronous code:
 
-* Overriding a limited set of methods,
+  {% prettify dart tag=pre+code %}
+  [!runZonedGuarded(() {!]
+    HttpServer.bind('0.0.0.0', port).then((server) {
+      server.listen(staticFiles.serveRequest);
+    });
+  [!},
+  (error, stackTrace) => print('Oh noes! $error $stackTrace'));!]
+  {% endprettify %}
+  
+  Running the HTTP server in a zone
+  enables the app to continue running despite uncaught (but non-fatal)
+  errors in the server's asynchronous code.
+
+* **Associating data**—known as
+  <em>zone-local values</em>—**with individual zones**.
+
+* **Overriding a limited set of methods**,
   such as `print()` and `scheduleMicrotask()`,
   within part or all of the code.
 
-* Performing an operation—such as
+* **Performing an operation**—such as
   starting or stopping a timer,
-  or saving a stack trace—each time that
-  code enters or exits a zone.
+  or saving a stack trace—**each time that
+  code enters or exits a zone**.
 
 You might have encountered something similar to zones in other languages.
 _Domains_ in Node.js were an inspiration for Dart’s zones.
```

Comment:
**question:** Why does this page explain the basics of zones this far down in the page (line 77)? Could and should this not appear further up?

Reply:
I don't think this is really _that_ far down. It's just at the bottom of the intro section. It's still introduction material, but not more important than the two parts the precede it: first paragraph stating what the article covers and prereqs, and second paragraph/block stating what zones are capable of.